blockbase/sass/blocks/_post-comments.scss

```diff
@@ -12,14 +12,52 @@
 		@include button-hover-styles;
 	}
 
-	.reply a {
-		display: inline-block;
+	.reply {
+		margin-block: var(--wp--custom--margin--vertical);
+
+		a {
+			display: inline-block;
+		}
 	}
 
-	.comment-form {
+	form {
 		input[type="text"] {
 			width: 100%;
 		}
+
+		input:not([type=submit]):not([type=checkbox]),
+		textarea {
+			border: var(--wp--custom--form--border--width) var(--wp--custom--form--border--style) var(--wp--custom--form--border--color);
+		}
+
+		.comment-form-cookies-consent {
+			input[type="checkbox"] {
```

Comment:
This needs the `@include hide-accesibly;` mixin to hide the original checkbox too